### DISCUSSION TITLE
A way to type the brightness and contrast without sliders

### DISCUSSION THREAD

**Discussion by DFarish** [arkypita/LaserGRBL#1642] [Ideas]
I would like to see where the settings for brightness and contrast numbers can be typed and not have to use a slider. It would be easier to set a number one at a time.
thanks,
Doug

**Comment by StuartB4** [comment]
Theres not anywhere to add numbers. 
But you can click on the slider then use the left and right arrow keys on the keyboard to change the numbers.

**Comment by Tribo-2dot0** [comment]
and if you double click the slider, it resets to 100 again.
